docs/UserGuide.md

```diff
@@ -494,11 +491,11 @@ the data of your previous UniNurse home folder.
 | **Reverse undo command**        | `redo`                                                                                                                       |
 | **Exit**                        | `exit`                                                                                                                       |
 
-* Note that `TASK` is composed of `TASK_DESCRIPTION | DATE_AND_TIME | FREQUENCY`
-* `DATE_AND_TIME` should be in the format like 16-10-2022 1030
-* If no `DATE_AND_TIME` is provided, then a default date and time of 24 hours from creation will be set
-* FREQUENCY` is made of two components, the number of time periods in between recurring tasks and the specific time period, e.g 3 month or 2 week
-* The time periods for `FREQUENCY` are: `days`, `weeks`, `months` and `years`
-* `FREQUENCY` can be be empty if it's a one off task, however a `DATE_AND_TIME` must be provided inorder to set a recurring task.
-* Note that capitalization does not matter for frequency.
-* Note that both `DATE_AND_TIME` and `FREQUENCY` are optional fields, however to have a recurring task both of them are required, while you can have a normal task with date time not be recurring, i.e `FREQUENCY` not required
+* Note that `TASK` is composed of `TASK_DESCRIPTION | DATE TIME | INTERVAL TIME_PERIOD`
+* `DATE TIME` should be in the format like `16-10-2022 1030` or `2-3-2022` if time is to be omitted
+* If no `DATE TIME` is provided, then a default date and time of 24 hours from creation will be set
+* The values for `TIME_PERIOD` are: `days`, `weeks`, `months` and `years`
+* If only no `TIME` is provided then the default time of 0000 hours will be set
+* Examples of valid `INTERVAL TIME_PERIOD` are: `3 days`, `7 weeks`, `2 months`
+* `INTERVAL TIME_PERIOD` can be be empty if it's a one off task, however a `DATE TIME` must be provided in addition to `INTERVAL TIME_PERIOD` inorder to set a recurring task.
+* Note that capitalization does not matter for `TIME_PERIOD`.
```

Comment:
I think it is better to move this to task params (and some appear to be repeated). This command summary is targeted at intermediate users of the app, so we could assume they roughly know these details